docs/dev-guide/guide.md

```diff
@@ -228,6 +228,53 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 - For each variant of a question that a student sees they will have submitted zero or more `submissions` with a `variant_id` to show what it belongs to. The submissions row also contains information the submitted answer and whether it was correct.
 
+??? tip "Schema visualization"
+
+    Using the `ms-ossdata.vscode-pgsql` extension, you can visualize the schemas within VSCode.
+
+    ![Setup Postgres VSCode](./postgres-vscode-setup.png)
+    ![Using Postgres VSCode](./postgres-vscode-tool.png)
+
+??? tip "Removing old schemas"
+
+    The following query will remove all schemas except public and the most recent one.
```

Comment:
This is clever, but is probably unnecessarily complex. Maybe we should instead document a script that removes everything but `public` and advise folks to just stop their server before cleaning and start it again after.

Reply:
Complex as in we might have to change it in the future? If you think that is the case, I'll update the recommendation.

Reply:
Complex as in inherently complex for not much value: there is no point in retaining even the latest schema in anything besides production environments, as restarting the server will create a new one automatically.